### PR TITLE
test/run: Add scenario for running integration tests with the Python bridge

### PR DESCRIPTION
**Pull request by martinpitt**
SCNR!

 - https://github.com/cockpit-project/bots/pull/3812

### CODE REVIEW

**Comment by martinpitt** [comment]
So of course that first run is a complete disaster. Logging in fails with "The cockpit package is not installed" (also happens locally). I can run the bridge just fine in the VM, so it's not just "missing python" or so. No SELinux either. No error message in journal.

The reason is that permissions are 744, which is of course broken. We need to fix that in our build system, added a separate commit.

**Comment by martinpitt** [comment]
Seems the next most important thing is to get the root bridge working. Right now there is no privilege indicator nor shell API to get to privileged mode. @allisonkarlitskaya, @mvollmer  would that be part of the ["routing" jira card](https://issues.redhat.com/browse/COCKPIT-866), or something else?

**Comment by mvollmer** [comment]
> Seems the next most important thing is to get the root bridge working.

We can also let the tests log in as root, I guess.  (Just for the python bridge and only until superuser stuff works, of course.)

> would that be part of the ["routing" jira card](https://issues.redhat.com/browse/COCKPIT-866), or something else?

The superuser API is implemented in the router, but maybe it makes sense to make it a separte jira issue.

**Comment by martinpitt** [comment]
Rebased to current main. I suggest to land this now -- we *know* the tests are horribly broken, but having the scenario available is useful for PRs like #17739 or @allisonkarlitskaya 's current bridge d-bus work, to compare progress.

File: src/Makefile.am [hunk 1]
```diff
@@ -90,6 +90,7 @@ cockpit-bridge.pyz: $(MANIFESTS) $(PYTHON_BRIDGE_FILES) $(SYSTEMD_CTYPES_STAMP)
 	@mkdir -p tmp/pyz
 	@cp -r $(srcdir)/dist $(srcdir)/src/cockpit src/systemd_ctypes/ tmp/pyz
 	$(AM_V_GEN) python3 -m zipapp --python /usr/bin/python3 --output $@ --main cockpit.bridge:main tmp/pyz
+	chmod a+x $@
```
[the remark this answers is not in the document]
Comment (martinpitt) [reply src/Makefile.am]: Fixed

**Review by allisonkarlitskaya — APPROVED** [review]
Looks good, thanks.  Looking forward to seeing how awfully broken this is :)

**Comment by martinpitt** [comment]
@allisonkarlitskaya : I specifically did *not* trigger the /pybridge test here, as I don't want to land a red PR. See the [previous test run](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17705-20220902-130642-97b858ef-fedora-36-devel/log.html) for an example.

**Comment by martinpitt** [comment]
Seems packit is completely b0rked right now.. But also, [this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17705-20220916-151134-e258e1fd-fedora-36-pybridge/log.html) is completely ridiculous right now.

**Pull request merged**